test/redfish-core/include/redfish_aggregator_test.cpp

```diff
@@ -1,3 +1,4 @@
+#ifdef BMCWEB_ENABLE_REDFISH_AGGREGATION
```

Comment:
Why?

Reply:
this test code pulls redfish-aggregator file and aggregator code is getting compiled and throws errors 
as aggregator code is implemented based on upstream http_client code.
if we don't enable BMCWEB_ENABLE_REDFISH_AGGREGATION then what's the point in running aggregator unit test code.

Reply:
We will enable BMCWEB_ENABLE_REDFISH_AGGREGATION  for future work

Reply:
yeah I understand, we will need to figure out how to fix this